components/dashboards-web-component/src/listing/components/DashboardCard.jsx

```diff
@@ -21,12 +21,16 @@ import { Link, withRouter } from 'react-router-dom';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 
-import { Card, CardMedia, CardTitle, Dialog, FlatButton, Menu, MenuItem, Popover, Snackbar } from 'material-ui';
+import {
+    Card, CardMedia, CardTitle, Dialog, FlatButton, Menu, MenuItem, Popover, Snackbar,
+    TextField
```

Comment:
Formatting issue? TextField can also go into the same line?

Reply:
Fixed